### PR TITLE
Add basic boostrap styling

### PR DESCRIPTION
**Pull request by sarahquigley**
- Add basic bootstrap styling.
- Configure CKEditor plugin to support insertion of 'Readmore' links, and to support a default class on images.
- Create grunt support for compiling admin script assets separately from other static assets.

### CODE REVIEW

**Comment by sarahquigley** [comment]
@mstewart Take a look and let me know what you think.

File: Gruntfile.js [hunk 1]
```diff
@@ -52,12 +55,22 @@ module.exports = function(grunt) {
...
       dev: {
         files: {
           '<%= config.dirs.dev %>/main.js': config.files.scripts
         }
-      }
+      },
+      admin: {
+        files: [{
+          expand: true,
+          cwd: 'ckeditor_custom/<%= config.dirs.static %>',
```
Comment (mstewart) [inline Gruntfile.js]: General pattern for dealing with admin scripts here seems good, but does the `ckeditor_custom` need to be hardcoded in here?

I.e. if there were multiple admin plugins, would it work to have a `cwd` of `*/<%= config.dirs.static %>` and allow the work of filtering that down to just the admin plugins to be done by the `.admin.coffee` filter?  Or failing that, have a configured list of admin plugin directories?

**Comment by mstewart** [comment]
Seems great!  You've definitely made the best of the nasty constraints put on you by this CKEditor business...

File: Gruntfile.js (hunk 1 above)
Comment (sarahquigley) [inline Gruntfile.js]: Some quick research indicates that there is no easy way around this:
- Th `cwd` attribute does not support arrays, or globbing patterns.
- Should you wish assets to be compiled in bulk relative (but not including) a specific path, the only way to do so is using the `cwd` attribute.

As far as I can see, the best I can do in future, is list out admin files as follows:

```
 admin: {
    files: [{
      expand: true,
      cwd: 'ckeditor_custom/<%= config.dirs.static %>',
      src: ['**/*.admin.coffee'],
      dest: '<%= config.dirs.dev %>',
      ext: '.admin.js',
    },{
      expand: true,
      cwd: 'some_app/<%= config.dirs.static %>',
      src: ['**/*.admin.coffee'],
      dest: '<%= config.dirs.dev %>',
      ext: '.admin.js',
    }]
},
```

I guess if you started to get fancy you could create some helper functions, for creating more complex config for grunt tasks- config such as this, that can't be created easily (so far as I can see) using their in-built options.

**Comment by sarahquigley** [comment]
Thanks for reviewing Mike! Glad you approve! :tomato:

**Pull request merged**